lib/service/commands/configure.rb

```diff
@@ -67,6 +85,21 @@ def data
         @data ||= load
       end
 
+      def config_input
+        if options.config == '@-'
+          $stdin.read
```

Comment:
Can we support `/dev/stdin` as another spelling of STDIN please?

Reply:
https://github.com/openflighthpc/flight-service/pull/8/commits/48a86a870bd0c629b17fb7bc254bd600b0f4d067